python/ray/train/huggingface/transformers/transformers_predictor.py

```diff
@@ -282,12 +281,6 @@ def _predict_pandas(
         feature_columns: Optional[List[str]] = None,
         **pipeline_call_kwargs,
     ) -> "pd.DataFrame":
-        if TENSOR_COLUMN_NAME in data:
-            arr = data[TENSOR_COLUMN_NAME].to_numpy()
-            if feature_columns:
-                data = pd.DataFrame(arr[:, feature_columns])
-        elif feature_columns:
-            data = data[feature_columns]
```

Comment:
Do we still need this?

Reply:
The line right below it says 
>      data = data[feature_columns] if feature_columns else data

both result in the same thing, so no I don't think we need it.